WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -855,73 +856,91 @@ public void onConfigurationChanged(Configuration newConfig) {
         }
     }
 
-    private String getSaveButtonText() {
+    private enum PrimaryAction {
+        SUBMIT_FOR_REVIEW,
+        SCHEDULE,
+        UPDATE,
+        SAVE
+    }
+
+    private enum SecondaryAction {
+        SAVE_AS_DRAFT,
+        PUBLISH_NOW,
+        NONE
+    }
+
+    private PrimaryAction getPrimaryAction() {
         if (!userCanPublishPosts()) {
-            return getString(R.string.submit_for_review);
+            return PrimaryAction.SUBMIT_FOR_REVIEW;
+        }
+
+        if (mPost.isLocalDraft()) {
```

Comment:
I just realized that a local draft can have a different post status:
- create a new post
- update the post status to "Private"
- we shouldn't offer the "Publish Now" option as the secondary action.

Since posts are created with the `Draft` status by default, I think it's fine to remove this test. I'm just wondering if we should differentiate "first" actions from other actions ("Update" seems weird for a Private post you just created)

Reply:
Removed in e1feffb